src/pages/Auth/SignupPage/Steps/ExternalDetailStep/index.tsx

```diff
@@ -131,7 +133,15 @@ function ExternalDetail({ onNext }: ExternalDetailStepProps) {
   return (
     <div className={styles.container}>
       <div className={styles.container__wrapper}>
-        <h1 className={styles.container__title}>회원가입</h1>
+        <div className={styles['container__title-wrapper']}>
+          <button type="button" onClick={onBack} aria-label="뒤로가기">
+            <BackIcon />
+          </button>
+          <h1 className={styles['container__title-wrapper--title']}>회원가입</h1>
+          <div className={styles['container__title-wrapper--icon']}>
```

Comment:
BEM 형식에 조금 어긋나는 것 같아요. modifier는 block이나 element의 속성을 나타내는데, 여기서는 그렇게 사용하지 않는 것 같아요.
```suggestion
          <h1 className={styles['container__title']}>회원가입</h1>
          <div className={styles['container__title-icon']}>
```
이런식으로만 써도 괜찮을 거 같아요!

Reply:
PR에 올려주신 동영상에서 확인했는데, icon이 필요없어 보이는 것 같아요. 왜 해당 element를 남겨두셨나요??

Reply:
동일 라인에 있는 상황에서 텍스트는 가운데, 아이콘은 좌측 정렬을 수행해야 했기에 **크기가 같고 보이지 않는 요소**를 추가해서 정렬을 수행했었습니다.
필요없는 icon이 맞고, 버튼에 `position: absolute;` 를 주는 방식이 더 깔끔해 보이길래 해당 방법으로 수정하였습니다!
수정하면서 네이밍 방식도 변경하였습니다!
감사합니다 😄